packages/template-retail-react-app/package.json

```diff
@@ -13,7 +13,7 @@
     "build-translations": "npm run extract-default-translations && npm run compile-translations",
     "compile-translations": "formatjs compile-folder --ast app/translations app/translations/compiled",
     "compile-translations:pseudo": "formatjs compile --ast app/translations/en-US.json --out-file app/translations/compiled/en-XB.json --pseudo-locale en-XB",
-    "extract-default-translations": "formatjs extract 'app/**/*.{js,jsx}' --out-file app/translations/en-US.json --id-interpolation-pattern [sha512:contenthash:base64:6]",
+    "extract-default-translations": "node ./scripts/build-translations.js",
```

Comment:
As you previously mentioned, this command is located in different places depending if you are an extended app or a base template. Because the way the generator works, it simply grabs all the scripts from the base template and places them in the package.json handlebars template, it would be difficult to dynamically change that path to the script. 

You've recommend moving the script to the sdk so we don't have this problem or we can use some dynamic execution logic inline. 

We should discuss this and implement one of them.

Reply:
an alternative is to put the script in a `bin` folder and declare the bin in package.json, then you could just do 

```json
{
  scripts: {
    "build-translations": "build-translation"
  },
  bin: {
    "build-translations": "./scripts/build-translations.js"
  }
}
```